Source/Scene/ParticleSystem.js

```diff
@@ -817,7 +1009,25 @@ define([
     };
 
     /**
-     * A function used to apply a force to the particle on each time step.
+     * A function used to modify attributes of the particle at each time step. This can include force modifications,
+     * color, sizing, etc.
+     * @callback ParticleSystem~updateCallback
+     *
+     * @param {Particle} particle The particle being updated.
+     * @param {Number} dt The time since the last update.
+     *
+     * @example
+     * function applyGravity(particle, dt) {
```

Comment:
This is nitpicky, but since we're not using an array of forces any more, would it make more sense to just call this function `updateParticle` instead of `applyGravity`? Up to you.

Reply:
i was thinking of leaving it as applyGravity just as a more descriptive example and demoing that they can call the method whatever they wish... However - if there's a name other than applyGravity that you feel works better - such as applyGravityForce or something - pls lmk. I just dont think updateParticle is descriptive enough for the docs.